MonumentAddons.cs

```diff
@@ -143,6 +145,7 @@ private void OnServerInitialized()
 
             _uniqueNameRegistry.OnServerInitialized();
             _adapterListenerManager.OnServerInitialized();
+            _monumentHelper.OnServerInitialized(_pluginInstance);
```

Comment:
Replace `_pluginInstance` with `this`.

Reply:
removed argument from `OnServerInitialized` and resolving dependency in plugin constructor